style.css

```diff
@@ -0,0 +1,118 @@
+section {
+  height: 718px;
+  width: 375px;
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-position: 0;
+}
+
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.toolbar .logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: 1 1 40%;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 20px;
+  padding-top: 10px;
+}
+
+.toolbar img {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-right: 20px;
+  padding-top: 10px;
+}
```

Comment:
Please know that according the Figma design, you need to have some margins set in your CSS, for example for the logo, you should have this set up:

![image](https://user-images.githubusercontent.com/20567503/176563788-d412242d-4865-45ce-8f03-907d278594c0.png)

This should help make your design look better! 😎😎